src/site/stages/build/drupal/graphql/GetLatestPageById.graphql.js

```diff
@@ -104,8 +105,9 @@ module.exports = `
     ${menuLinksQuery}
   }
 `;
+};
 
-const query = module.exports;
+const query = buildQuery();
```

Comment:
```suggestion
const query = await buildQuery();
```

Reply:
`buildQuery` is already `await`-ing the call to `facilitySidebarQuery.compiledQuery`. There is no need to `await`here, and in fact doing so would cause a parse error.

Reply:
Since it's an `async` function now, it's going to return the string wrapped in a Promise.

If you want to handle `query` as a string and not as a string wrapped in a Promise, you should `await` here.

Reply:
I suppose the issue is then that that part is not wrapped in an `async` function itself, so you can't `await`.

Reply:
So as it is now, you'd get an error like `query.replace is not a function`.

What the fix will look like is for this module to return something like `query.then(q => q.replace(...)`.

`getQuery`, which requires this module, is already treated like an async function in some places. But as I mentioned in another comment, you'll want to `await` its results in the other places where it's being used (like `getLatestPageById` in`drupal/api.js`).

Reply:
OK @U-DON - I've made the changes you suggested.

It looks like `getLatestPageById` is only used by the preview server. 
However, I do not see the side navs when I try to preview facility pages on master, so I think this is an existing bug.